src/MinHeap.test.js

```diff
@@ -0,0 +1,114 @@
+var MinHeap = require('./MinHeap');
+
+describe('MinHeap', function() {
+  describe('#getFirst', function() {
+    describe('when initialized', function() {
+      it('first value is undefined', function() {
+        var heap = MinHeap();
+        expect(heap.getFirst()).toBeUndefined();
+      });
+    });
+
+    describe('after initialization', function() {
+      describe('when a value is added', function() {
+        it('first value is added value', function() {
+          var heap = MinHeap();
+          heap.add(1);
+          expect(heap.getFirst()).toBe(1);
+        });
+
+        describe('when the value is removed', function() {
+          it('first value is undefined', function() {
+            var heap = MinHeap();
+            heap.add(1);
+            heap.remove();
+            expect(heap.getFirst()).toBeUndefined();
+          });
+        });
+      });
+    });
+  });
+
+  describe('#add', function() {
+    describe('when empty', function() {
+      it('value is first value', function() {
+        var heap = MinHeap();
+        heap.add(1);
+        expect(heap.getFirst()).toBe(1);
+      });
+
+      describe('add smaller value', function() {
+        it('smaller value is first value', function() {
+          var heap = MinHeap();
+          heap.add(1);
+          heap.add(0);
+          expect(heap.getFirst()).toBe(0);
+        });
+      });
+
+      describe('add larger value', function() {
+        it('initial value is first value', function() {
```

Comment:
Wrong description: `smaller value is first value`?
Since it shouldn't matter the order in which you add values.